config/application.rb

```diff
@@ -1,22 +1,34 @@
 require_relative "boot"
 
 require "rails"
-
+# Pick the frameworks you want:
+require "active_model/railtie"
+require "active_job/railtie"
+# require "active_record/railtie"
+# require "active_storage/engine"
 require "action_controller/railtie"
+# require "action_mailer/railtie"
+# require "action_mailbox/engine"
+# require "action_text/engine"
+require "action_view/railtie"
+# require "action_cable/engine"
+require "rails/test_unit/railtie"
```

Comment:
Do we need these extra frameworks? I note that we don't use ActiveJob, Views or test_unit

Reply:
Fixed.